src/hooks/sound/useSoundCreateInputs.tsx

```diff
@@ -43,14 +44,16 @@ const useSoundCreateInputs = () => {
 
     const allocations = getSoundSplitAllocations(splitArgs)
     const splitWallet = await getSplitWalletV1(splitArgs)
+    const isDeployedSplit = await isDeployedSplitWalletV1(splitArgs)
 
-    const createSplitConfig = shouldSplit
-      ? {
-          distributorFee: DEFAULT_DISTRIBUTOR_FEE,
-          controller: zeroAddress,
-          accountAllocations: allocations,
-        }
-      : null
+    const createSplitConfig =
+      shouldSplit && !isDeployedSplit
```

Comment:
could add `!isDeployedSplit` to the declaration of `shouldSplit` right?
- less logic in the ternary